setup.cfg

```diff
@@ -1,17 +1,15 @@
 [bumpversion]
-# Do not edit this string manually, always use bumpversion
-# Details in CONTRIBUTING.md
-current_version = 3.0.0
+current_version = 3.0.2
```

Comment:
you set this here and in the __init__.py is there a way it could just be in one place?

Reply:
Unfortunately no. But if you read Contributing.... all places this is used is adjusted by running `bumpversion` this isn't manually set.